data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,469 @@ mission "Coalition License Patch"
+			choice
+				`	"Sounds like you're all really excited about this. Alright, I'll bring you there."`
+					goto bbbb
+				`	"Well I wish you luck then, but sadly I'm not looking for any transport jobs right now."`
+					decline
+			label bbbb
+			`	"Most grateful, we are," the Saryd says.`
+			`	As you help them bring their luggage into your ship, they introduce themselves: the Saryd is called Ulmodi, the Kimek is Pairri, and the Arach is Tumub. They settle in their bunks to rest for the upcoming event, though as soon as Pairri closes her bunk's door you hear some slight noises. It sounds like she's doing push-ups - or maybe squats; you can't really tell how a Kimek would name such exercises.`
+			`	Ulmodi explained that they need to get to <destination> (by <date>). It shouldn't be a problem considering you even have a Jump Drive to get there faster.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 300000
+
+
+
+mission "Coalition Games 2"
+	invisible
+	landing
+	to offer
+		has "Spoder Bootle Centoro Athletes: done"
+	source "Far Home"
+	on offer
+		conversation
+			`As the athletes leave your ship and hand you your pay of 300,000 credits, Pairri stretches her legs, trying to get eye-level with you.`
+			`	You crouch, and she says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."`
+			`	She extends one of her legs to you, holding a small, coin-like piece. It has a symbol like the old Olympic Games on Earth, except with three interlocked rings instead of five.`
+			`	"Very long and tiring, for a first timer the games can be. Reading an entire novel in an hour, it is like," Tumub explains. "Understand, we would, if too busy to watch them, you are."`
+			`	"Still, much joy, it would bring us, if to watch the games, you chose," says Ulmodi. "Rare, such a chance is, to show an outsider the wonders of our games."`
+			choice
+				`	"Thank you. I would be glad to view the games."`
+				`	"I'm honored, but sadly I have other matters to attend to right now."`
+					decline
+			`	You take the 'ticket', and they start explaining how the games work.`
+			`	"Separate competitions, each species has," Tumub says. "To value each one's history, it is so."`
+			`	"A grand, cooperative match, at the end of it all takes place," Ulmodi further details. "Teams consisting of all three species, formed they are. A joyous, unmatched experience, the last spectacle is."`
+			`	Since they need to leave to prepare for the games, they can't tell you about each sport but assure you that you'll like them. You find a Heliarch agent, and they quickly point you to the line where people are checking in with their own tickets. The line must be thousands long, but you barely spend ten minutes in it. The Heliarchs taking care of admissions are incredibly efficient and keep things organized.`
```

Comment:
```suggestion
			`	Since they need to leave to prepare for the games, they can't tell you about each sport, but assure you that you'll like them. You find a Heliarch agent who quickly points you to the line where people are checking in with their own tickets. The line must be thousands long, but you barely spend ten minutes in it. The Heliarchs taking care of admissions are incredibly efficient and keep things organized.`
```